docs/src/overview/readme.stories.mdx

```diff
@@ -35,6 +35,10 @@ import {Meta} from '@storybook/addon-docs/blocks';
 
 We love [react-hooks](https://code.fb.com/open-source/react-hooks) and willing to share our [react-native](https://facebook.github.io/react-native) ui components built in [function components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) using hooks. Also the `dooboo-ui` is built on top of our favorite stacks like [emotion](https://emotion.sh/docs/@emotion/native), [typescript](https://www.typescriptlang.org/), [jest](https://jestjs.io), [react-testing-library](https://testing-library.com/docs/react-testing-library/intro), [expo](https://expo.io), [storybook](https://storybook.js.org), and so on.
 
+## Philosophy
+
+We are aiming to making `dooboo-ui` to be more user friendly and customizable, powerful library.
```

Comment:
I think you might want to change this sentence more fluent.
`dooboo-ui` aims to provide easy, lightweight, and flexible UI components?

Reply:
oh, thank you for comment! i will fix it.